app/Resources/translations/default/ModulesBlockwishlistShop.xlf

```diff
@@ -75,27 +75,42 @@
       <trans-unit id="f11a5449691a6083161a98a08a549541">
         <source>Unable to update product from wishlist</source>
         <target>Unable to update product from wishlist</target>
-        <note>Line: 270</note>
+        <note>Line: 294</note>
       </trans-unit>
       <trans-unit id="331d764872260ecaec822a6768871e97">
         <source>Request is missing one or multiple parameters</source>
         <target>Request is missing one or multiple parameters</target>
-        <note>Line: 320</note>
+        <note>Line: 344</note>
       </trans-unit>
       <trans-unit id="6354c00dadee375467a1b8bddfdc6aaa">
         <source>Product added to cart</source>
         <target>Product added to cart</target>
-        <note>Line: 348</note>
+        <note>Line: 373</note>
       </trans-unit>
       <trans-unit id="5fde6bcb72b2b1f2a179fc2a62094f80">
         <source>Error when adding product to cart</source>
         <target>Error when adding product to cart</target>
-        <note>Line: 355</note>
+        <note>Line: 381</note>
       </trans-unit>
       <trans-unit id="3a9037f4f377fb6ff22367d375897faa">
         <source>You're not allowed to manage this list.</source>
         <target>You're not allowed to manage this list.</target>
-        <note>Line: 387</note>
+        <note>Line: 412</note>
+      </trans-unit>
+      <trans-unit id="3c8d281dbe07e2372c1c8413859cdfd0">
+        <source>There was an error adding the product attributes</source>
```

Comment:
there was an error **while** adding (...)

Reply:
Mmm this one is in the module wishlist, I'm gonna open a PR to add it but I'm not sure it will be released soon enough

Reply:
https://github.com/PrestaShop/blockwishlist/pull/115